docs/EXTERNAL_APP_QUICKSTART.md

```
@@ -0,0 +1,447 @@
+- **Port conflict:** If port 5565 is in use, map to a different port: `docker run -p 6000:5565 ...` and update your URI to `ws://localhost:6000`.
+
+### Authentication errors
+
+- Verify your API key is correct and matches what the engine expects.
+- Check that the `ROCKETRIDE_APIKEY` environment variable is set if you are not passing `auth` directly.
+- In persist mode, `AuthenticationException` stops retries so you can fix credentials and call `connect()` again.
+
+### Pipeline file not found
+
+- `use(filepath=...)` resolves paths relative to the engine's working directory, not your application. For Docker deployments, mount the pipeline file into the container:
+  ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)